<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ABHICHAK-LAP" name="Mod2" tests="4" failures="1" timestamp="8 Jan 2022 13:40:56 GMT" time="0.004" errors="0">
  <testcase name="NEFT" time="0.0" classname="learntestng.MoneyTransfer"/>
  <testcase name="IMPS" time="0.0" classname="learntestng.MoneyTransfer"/>
  <testcase name="RTGS" time="0.001" classname="learntestng.MoneyTransfer">
    <failure type="java.lang.AssertionError" message="expected [1] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [2]
	at learntestng.MoneyTransfer.RTGS(MoneyTransfer.java:45)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- RTGS -->
  <testcase name="Gpay" time="0.0" classname="learntestng.MoneyTransfer">
    <skipped/>
  </testcase> <!-- Gpay -->
</testsuite> <!-- Mod2 -->
